addons/sms/wizard/sms_composer_views.xml

```diff
@@ -43,7 +43,7 @@
                             attrs="{'invisible': [('comment_single_recipient', '=', False)]}"/>
                         <div attrs="{'invisible': [('comment_single_recipient', '=', False)]}">
                             <field name="recipient_single_description" class="oe_inline"/>
-                            <field name="recipient_single_number_itf" class="oe_inline" nolabel="1" options="{'onchange_on_keydown': True}" placeholder="e.g. +1 415 555 0100"/>
+                            <field name="recipient_single_number_itf" class="oe_inline" nolabel="1" onchange_on_keydown="true" placeholder="e.g. +1 415 555 0100"/>
```

Comment:
picky std, because it's a python view I would use `True` (even it does exactly the same)